NEWS.md

```diff
@@ -1,3 +1,23 @@
+# tidycomm 0.3.0
+
+## New features
+* Converted `tibble` returns into `tdcmm/tibble` return objects (they behave just like tibbles but are in essence our own objects now)
+* Added partial correlation in `correlate(..., partial = TRUE)`
+* Added linear regression `regress()`
+* Added one-sample t-test `t_test(..., my = ...)`
+* Added `reverse_scale()`, `minmax_scale()`, `z_scale()`, and `center_scale()` to shift and modify continuous scales
+* Added `tab_percentiles()`
+* Added `visualize()` to visualize almost everything
+* Added `snscomments` and `incvlcomments` as additional data sets
+
+## Minor changes
+* Changed `unianova()` and `t_test()` to build on `leveneTest()`
+* Allowed `test_icr()` to work with grouped data
+* Converted all code examples in documentation to be built on `tidycomm`-provided data sets

```

Comment:
Might be a good idea to mention the updated tibble column names for the unianova posthoc test. I've renamed the columns, e.g.: "estimates" --> "Delta_M" to match Games-Howell and Tukey's HSD. This might break scripts if colleagues have relied on these names.